src/ec/packages/ec-component-footer/__snapshots__/footer.test.js.snap

```diff
@@ -48,20 +48,28 @@ exports[`EC - Footer Corporate renders correctly 1`] = `
               class="ecl-footer__section-item"
             >
               <a
-                class="ecl-link ecl-link--standalone ecl-footer__section-link"
+                class="ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon- ecl-footer__section-link"
```

Comment:
When there is no icon, link should remain with a simple structure (no nested span)

Reply:
Mmhh..yes but then why do we call the link component passing an icon if we don't need it?
The way the icon is passed here (only the size) tricks the logic in the link component. 
The thing here is: what should be the miminum requirement for outputting an icon in the link component? I think we had a check for the icon name before, i can set it back, but still is questionable the fact that we pass data we don't need, i think.

Reply:
Indeed we did a test on the icon name. I don't have better idea for this, so I guess we could just stick to this test.
The issue with the footer, it that icon size are enforced, to looks like ECL's footer. We could indeed set the icon size in the data file, but in this case that would be up to the integrator to set the correct size (with potential errors)